src/constants.jl

```diff
@@ -88,3 +88,57 @@ const global ipums_sources = [
         collection_type = "microdata"
     )
 ]
+
+
+#=
+
+XPaths for parsing DDI XML files
+
+This is a list of XPATHs for locating file-level and variable-level metadata 
+in a DDI XML file. The constants are stored here, but used in the `ddi_parser.jl`
+file.
+
+Extract level XPATHs:
+
+EXTRACT_CONDITIONS - IPUMs conditions for fair and legal use of public data.
+EXTRACT_CITATION - Citation information for referencing IPUMs data
+EXTRACT_IPUMS_PROJECT - Name of the IPUMs project from which the data is taken, such as CPS, or DHS, etc.
+EXTRACT_NOTES - User provided notes or additional miscellaneous information provided about the extract.
+EXTRACT_DATE - The date that the extract was generated.
+
+Variable level XPATHs:
+
+VAR_NAME_XPATH = The name of the IPUMs variable.
+VAR_STARTPOS_XPATH = The start position (in text columns) of the variable in a fixed width file specification.
+VAR_ENDPOS_XPATH = The end position (in text columns) of the variable in a fixed width file specification.
+VAR_WIDTH_XPATH = The width postion (in text columns) of variable in a fixed width file specification. 
+VAR_LABL_XPATH = A short description for the data contained in a variable.
+VAR_TXT_XPATH = A longer and more complete description of the data contained in a variable.
+VAR_DCML_XPATH = The number of decimal points contained in a variable.
+VAR_TYPE_XPATH = An indicator of whether the variable is either a string or numeric data type.
+VAR_INTERVAL_XPATH = An indicator of whether a numeric variable is continuous or discrete.
+VAR_CATEGORY_XPATH = A description of the category levels and corresponding numerical values for a categorical variable, such as 
+                        "Women => 0, Men => 1"
+
+=#
+
+EXTRACT_CONDITIONS = "/x:codeBook/x:stdyDscr/x:dataAccs/x:useStmt/x:conditions"
```

Comment:
these should be `const`s before merging for speed

Reply:
Yep, I have that updated now. Now all of these have `const` in front of them.